eng/templates/stages/release.yml

```diff
@@ -401,6 +405,8 @@ stages:
             echo "  --title $title"
             echo "  --body $body"
             echo "  --targetBranch $target_branch"
+            echo "  --resolvedSdkArtifactFileName $resolvedSdkArtifactFileName"
+            echo "  --resolvedSourceBuiltArtifactsFileName $resolvedSourceBuiltArtifactsFileName"
```

Comment:
I think this is no longer `--resolved..`

Reply:
There is already a variable called sourceBuiltArtifactsFileName.

Reply:
Yeah but we need to fix it here too?

```suggestion
            echo "  --sdkArtifactFileName $resolvedSdkArtifactFileName"
            echo "  --sourceBuiltArtifactsFileName $resolvedSourceBuiltArtifactsFileName"
```

Reply:
Done, thank you :)